src/shared/lib/mutationUtils.ts

```diff
@@ -0,0 +1,5 @@
+import {GENETIC_PROFILE_UNCALLED_MUTATIONS_SUFFIX} from "../constants";
+
+export function isUncalled<T extends {geneticProfileId?:string}>(mutation:T) {
```

Comment:
agreed w @alisman, I'd say either make it nonoptional or make it accept the genetic profile id directly